google/cloud/security/common/data_access/sql_queries/create_tables.py

```diff
@@ -73,3 +73,12 @@
         PRIMARY KEY (`id`)
     ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 """
+
+CREATE_GSUITE_ACCOUNT_GROUPS_TABLE = """
```

Comment:
Agreed. I put some thought into this too.

Currently the API is defined here as the [Directory API](https://developers.google.com/admin-sdk/directory/v1/guides/manage-groups#get_all_domain_groups). When fetching groups you can scope the request to an account, a domain, or a sub-domain. Since it's my intention to specify account in the request I felt it appropriate to name the table with the same scope.

As it relates to GSuite I'm open to other ideas.